documentation/SDKs/javascript/README.md

```diff
@@ -0,0 +1,358 @@
+#JavaScript SDK for Nether
+
+## Work in Progress
+
+## Downloading the JavaScript SDK for Nether
+
+### Installing the JavaScript SDK for Nether
+
+## Usage
+
+### Analytics
+
+#### Active sessions - daily
+You can use the ***Nether.Analytics.GetActiveSessionsDaily*** method to get the total active sessions per day. Returns an array of ActiveSession
```

Comment:
There's a method for retrieving the info endpoint to use to send analytics events, and that should be used internally by the SDK to determine where to send the events. @AndyCross - did you share the JS code I sent you with @theciscokid ?